wire-ios/Wire-iOS/Sources/UserInterface/Settings/DatabaseStatisticsController.swift

```diff
@@ -80,8 +80,8 @@ final class DatabaseStatisticsController: UIViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        let session = ZMUserSession.shared()
-        let syncMoc = session!.managedObjectContext.zm_sync!
+        guard let session = ZMUserSession.shared() else { return }
```

Comment:
Do we want to add something like `assertionFailure` here to crash in debug builds?